resources/ext/cli_defaults/cli-defaults.sh.erb

```diff
@@ -1,10 +1,7 @@
 INSTALL_DIR="/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>"
 
-JRUBY_JAR="${JRUBY_JAR:-${INSTALL_DIR}/jruby-9k.jar}"
-
-if [ ! -e "$JRUBY_JAR" ]; then
-  echo "Unable to find specified JRUBY_JAR: ${JRUBY_JAR}" 1>&2
-  return 1
+if [ -n "$JRUBY_JAR" ]; then
+  echo "Warning: the JRUBY_JAR setting is no longer needed and will be ignored." 1>&2
```

Comment:
I wonder if it's also a good idea to log a warning on startup for puppet server if JRUBY_JAR has been set?

Reply:
I'm not 100% sure how it works, but this warning DOES get issued when puppetserver starts. It goes to the log.

Reply:
Oh... that's weird

Reply:
How'd you start Puppet Server? I'd expect this to go to stderr when running `puppetserver start` and when using a service that calls `puppetserver start` for it to go somewhere like systemd's journal.

Reply:
Oh I started it with `service puppetserver start`, and it did go to the journal. I bet with `puppetserver start` it would go to stderr.